README.md

```diff
@@ -2,53 +2,67 @@
 [![PyPI version](https://badge.fury.io/py/amundsen-search.svg)](https://badge.fury.io/py/amundsen-search)
 [![Build Status](https://api.travis-ci.com/lyft/amundsensearchlibrary.svg?branch=master)](https://travis-ci.com/lyft/amundsensearchlibrary)
 [![Coverage Status](https://img.shields.io/codecov/c/github/lyft/amundsensearchlibrary/master.svg)](https://codecov.io/github/lyft/amundsensearchlibrary?branch=master)
-[![License](http://img.shields.io/:license-Apache%202-blue.svg)](LICENSE)
+[![License](https://img.shields.io/:license-Apache%202-blue.svg)](LICENSE)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://bit.ly/2FVq37z)
 
 Amundsen Search service serves a Restful API and is responsible for searching metadata. The service leverages [Elasticsearch](https://www.elastic.co/products/elasticsearch "Elasticsearch") for most of it's search capabilites.
 
 For information about Amundsen and our other services, visit the [main repository](https://github.com/lyft/amundsen). Please also see our instructions for a [quick start](https://github.com/lyft/amundsen/blob/master/docs/installation.md#bootstrap-a-default-version-of-amundsen-using-docker) setup  of Amundsen with dummy data, and an [overview of the architecture](https://github.com/lyft/amundsen/blob/master/docs/architecture.md).
 
+## Requirements
+
+- Python >= 3.6
+
+
 ## Instructions to start the Search service from distribution
+
 ```bash
 $ venv_path=[path_for_virtual_environment]
-$ python3 -m virtualenv $venv_path
+$ python3 -m venv $venv_path
```

Comment:
the module name is virtualenv.

Reply:
You’re right it used to be that. But since python 3.3 `python -m venv` got introduced as a standard library module with PEP 405. See https://docs.python.org/3/library/venv.html

This also makes search on par with the metadata service state (and its README.md.

As covered in https://github.com/lyft/amundsen/issues/79 code in `amundsensearchlibrary` code already relies on 3.6.

Reply:
good to know. Thanks.